### DISCUSSION TITLE
v43.0

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#3991] [Release Discussions]
<h3>New Quests</h3>
  <ul>
  <li><i>How many steps are here?</i>, asked for stiles (#3932, #3141), by @matkoniecz</li>
  <li><i>Is there a working bicycle pump available for everyone here?</i>, asked for bicycle shops etc. (#3934, #3053), by @mnalis</li>
  <li><i>Is there an air compressor available here?</i>, asked for fuel stations (#3934, #3053), by @mnalis</li>
  <li><i>Are credit or debit cards accepted here?</i>, disabled by default (#3953), by @matkoniecz</li>
  </ul>
  <h3>Other Enhancements and Fixes</h3>
  <ul>
  <li>Do not ask questions about objects tagged as seasonal (#3943)</li>
  <li>Fix in rare cases answered sidewalk quest would not go away (#3976)</li>
  <li>Kerb type: Remove kerb if user answered that there is no kerb at all (#3983)</li>
  <li>Ask about opening hours and wheelchair accessibility for a few more kind of places (#3982), by @dbdean</li>
  <li>Better indicate when there is no GPS signal (#3944, #3977)</li>
  <li>Various minor enhancements and fixes (#3931, #3933, #3717, #3951, #3698, #3795, #3972, #3978, #3962, #2652...), thanks @mnalis, @CJ-Malone, @matkoniecz</li>
  </ul>
  <h3>New Translations</h3>
  <ul>
  <li>Added Armenian, by Alexander, Vasak87 and Anteneh Belayneh</li>
  <li>Added Esperanto, by Jolesh, Jan, Martin Constantino-Bodin, Hendursaga, Robin van der Vliet, Fabio and more</li>
  </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v43.0'>v43.0</a>.</em>

**Comment by KuboF** [comment]
Great to see #Esperanto here! Many thanks!

But in the program itself I do not see it in the language election. Any suggestions?

**Reply by FloEdelmann** [reply]
Indeed, the list for that setting was not updated. I added the new languages there now (https://github.com/streetcomplete/StreetComplete/commit/34fcef46c76b1209f5343edb9ee94ead6c5279b8), so they should be selectable in the next release.

Also, I mentioned it in the pre-release instructions (https://github.com/streetcomplete/StreetComplete/commit/0bff6242d5d0d0bf3f10cdef9110367a4b1f1406) to prevent this from happening again.
